dist/api_data.py

```diff
@@ -1518,9 +1518,17 @@ def __ge__(self, other):
         to the bit count (except for the last set of values loaded)'''),
     Config('checkpoint', '', r'''
         the name of a checkpoint to open (the reserved name
-        "WiredTigerCheckpoint" opens the most recent internal
+        "WiredTigerCheckpoint" opens the most recent
         checkpoint taken for the object).  The cursor does not
         support data modification'''),
+    Config('checkpoint_use_history', 'true', r'''
+        when opening a checkpoint cursor, open history store cursors and retrieve
+        snapshot and timestamp information from the checkpoint. This is in general
+        required for corect reads; if setting it to false the caller must ensure
```

Comment:
`corect` -> `correct` (not sure why s_all didn't catch this)?

Reply:
Fixed in c25a4ab. (Seems to not get caught because undoc=True; that keeps the text from landing anywhere that gets spellchecked.)